starboard/contrib/cast/cast_starboard_api/samples/BUILD.gn

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 The Cobalt Authors. All rights reserved.
+import("//starboard/build/config/starboard_target_type.gni")
+
+group("cast") {
+  if (build_with_separate_cobalt_toolchain) {
```

Comment:
Keep an eye on #1111  and #1181, maybe #1169 as well

Reply:
Thanks for the heads up; I believe I have these integrated already and they are not affecting the binary at the moment.